statsmodels/tsa/tests/test_tsa_tools.py

```diff
@@ -641,11 +641,17 @@ def test_duplicate_const(self):
             has_constant="raise",
         )
         df = pd.DataFrame(self.c)
-        assert_raises(
-            ValueError, tools.add_trend, x=df, trend="c", has_constant="raise"
+       assert_equal(
```

Comment:
This is really ancient code. 

Could you do something like

```python
with pytest.raises(ValueError, match="...."):
    tools.add_trend(x=df, trend="c", has_constant="raise")
```

where the test in `match`  has the string you are trying to test.

Same below with the `pytest.raises`